Sources/Web3Core/KeystoreManager/BIP32Keystore.swift

```diff
@@ -216,6 +216,54 @@ public class BIP32Keystore: AbstractKeystore {
         try encryptDataToStorage(password, data: serializedRootNode, aesMode: self.keystoreParams!.crypto.cipher)
     }
 
+    /// Fast generation addresses for current account
+    /// used for shows which address user will get when changed number of his wallet
+    /// - Parameters:
+    ///   - password: password of seed storage
+    ///   - number: number of wallets addresses needed to generate from 0 to number-1
```

Comment:
It confused me a little when you wrote `... from 0`. I got the impression that the `number` value could be `>=0`.

Could you please replace `from 0 to number-1` as this information is purely internal with `number must be greater than 0`.

```
number of addresses to generate. `number` must be greater than 0.
```

And I'm not completely sure that you'll always get back an array the length of which will be equal to the `number`.

Reply:
Perfect solution! 👍
I haven't thought of that.